commit 7205cbc5abda0a8571170a132bd30fc54a4aa6b6
Author: fritsch <peter.fruehberger@gmail.com>
Date:   Tue Jun 18 18:59:05 2013 +0200

    VPP: SkipDeint was not only meant for interlaced content (fixes Audio out of sync after pause, ffwd)

diff --git a/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp b/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp
index fbe3fd5..0f138aa 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp
+++ b/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp
@@ -771,8 +771,8 @@ void CVPPThread::Process()
     if(currentFrame.valid)
     {
       bool isInterlaced = currentFrame.DVDPic.iFlags & DVP_FLAG_INTERLACED;
-      if(currentFrame.DVDPic.iFlags & DVP_FLAG_DROPDEINT)
-        isInterlaced = false;
+      //if(currentFrame.DVDPic.iFlags & DVP_FLAG_DROPDEINT)
+      //  isInterlaced = false;
 
       EDEINTERLACEMODE   mode = g_settings.m_currentVideoSettings.m_DeinterlaceMode;
       EINTERLACEMETHOD method = g_settings.m_currentVideoSettings.m_InterlaceMethod;
diff --git a/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.h b/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.h
index 6ccef59..b7b82c9 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.h
+++ b/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.h
@@ -181,7 +181,7 @@ public:
   virtual void Close();
   virtual const std::string Name() { return "vaapi"; }
   virtual CCriticalSection* Section() { if(m_display) return m_display.get(); else return NULL; }
-  virtual bool CanSkipDeint() { return true; }
+  virtual bool CanSkipDeint() { return false; }
 
   int   GetBuffer(AVCodecContext *avctx, AVFrame *pic);
   void  RelBuffer(AVCodecContext *avctx, AVFrame *pic);
